ci-operator/step-registry/cucushift/aws/upi/cucushift-aws-upi-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: cucushift-aws-upi
+  steps:
+    pre:
+      - chain: ipi-aws-pre
```

Comment:
You are adding tests to UPI, thus your workflow should not be setting up IPI cluster